clients/lsp-xml.el

```diff
@@ -164,6 +170,7 @@ Newlines and excess whitespace are removed."
 
 (lsp-register-custom-settings '
  (("xml.validation.schema" lsp-xml-validation-schema t)
+  ("xml.validation.resolveExternalEntities" lsp-xml-validation-resolve-external-entities nil)
```

Comment:
the final `nil` is probably not needed

Reply:
Ah, thank you! fixed in 3f12e111 and rebased.